src/PersistenceStrategy/PostgresSimpleStreamStrategy.php

```diff
@@ -100,4 +100,16 @@ public function generateTableName(StreamName $streamName): string
 
         return $table;
     }
+
+    private function getSchemaCreationSchema(string $tableName) : array
```

Comment:
`getSchemaCreationSchema` sounds a bit off with those 2 schema in the name

Reply:
Sounds off. But according to the current naming scheme the SQL-statement that is sent to the database is called a scheme. And the structure that will be created in the database by this scheme is ... well... a scheme. So either all methods handling/creating SQL that shall be executed would need to be renamed or Postgres needs to rename their database-structure called schema. 

I could of course rename the function to `createDatabaseSchemaCreationSchema`. Not sure though that's better....

Is that a problem that prohibits merging the PR?

Reply:
@heiglandreas Thanks for the explanation. All good, we'll merge this PR soon. Just waiting for some feedback on the other issue.